From 28a1d145c82c6c75325dce58d9dcd53551320683 Mon Sep 17 00:00:00 2001
From: Michelle McDaniel <adiaaida@gmail.com>
Date: Tue, 6 Mar 2018 12:37:23 -0800
Subject: [PATCH] Disable Perf Runs on Commit (#16781)

We need to disable these jobs on commit until dotnet/core-eng#2808 is fixed.
---
 perf.groovy | 70 ++++++++++++++++++++++++++++++-------------------------------
 1 file changed, 35 insertions(+), 35 deletions(-)

diff --git a/perf.groovy b/perf.groovy
index ac471cb..db98bbf 100644
--- a/perf.groovy
+++ b/perf.groovy
@@ -133,15 +133,15 @@ def static getOSGroup(def os) {
                         builder.triggerForBranch(branch)
                         builder.emitTrigger(newJob)
                     }
-                    else if (opt_level == 'full_opt') {
+                    //else if (opt_level == 'full_opt') {
                         // Set a push trigger
-                        TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
-                        builder.emitTrigger(newJob)
-                    }
-                    else {
+                    //    TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
+                    //    builder.emitTrigger(newJob)
+                    //}
+                    //else {
                         // Set periodic trigger
-                        Utilities.addPeriodicTrigger(newJob, '@daily')
-                    }
+                    //    Utilities.addPeriodicTrigger(newJob, '@daily')
+                    //}
                 }
             }
         }
@@ -237,15 +237,15 @@ def static getOSGroup(def os) {
                             builder.triggerForBranch(branch)
                             builder.emitTrigger(newJob)
                         }
-                        else if (opt_level == 'full_opt' && pgo_optimized) {
+                        //else if (opt_level == 'full_opt' && pgo_optimized) {
                             // Set a push trigger
-                            TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
-                            builder.emitTrigger(newJob)
-                        }
-                        else {
-                            // Set periodic trigger
-                            Utilities.addPeriodicTrigger(newJob, '@daily')
-                        }
+                        //    TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
+                        //    builder.emitTrigger(newJob)
+                        //}
+                        //else {
+                        //    // Set periodic trigger
+                        //    Utilities.addPeriodicTrigger(newJob, '@daily')
+                        //}
                     }
                 }
             }
@@ -381,11 +381,11 @@ parallel(
         builder.triggerForBranch(branch)
         builder.emitTrigger(newFlowJob)
     }
-    else {
+    //else {
         // Set a push trigger
-        TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
-        builder.emitTrigger(newFlowJob)
-    }
+    //    TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
+    //    builder.emitTrigger(newFlowJob)
+    //}
 
 } // isPR
 
@@ -525,11 +525,11 @@ parallel(
         builder.triggerForBranch(branch)
         builder.emitTrigger(newFlowJob)
     }
-    else {
+    //else {
         // Set a push trigger
-        TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
-        builder.emitTrigger(newFlowJob)
-    }
+    //    TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
+    //    builder.emitTrigger(newFlowJob)
+    //}
 
 } // isPR
 
@@ -637,15 +637,15 @@ parallel(
                         builder.triggerForBranch(branch)
                         builder.emitTrigger(newJob)
                     }
-                    else if (opt_level == 'full_opt') {
+                    //else if (opt_level == 'full_opt') {
                         // Set a push trigger
-                        TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
-                        builder.emitTrigger(newJob)
-                    }
-                    else {
+                    //    TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
+                    //    builder.emitTrigger(newJob)
+                    //}
+                    //else {
                         // Set periodic trigger
-                        Utilities.addPeriodicTrigger(newJob, '@daily')
-                    }
+                    //    Utilities.addPeriodicTrigger(newJob, '@daily')
+                    //}
                 }
             }
         }
@@ -717,7 +717,7 @@ parallel(
         Utilities.standardJobSetup(newJob, project, false, "*/${branch}")
 
         // Set the cron job here.  We run nightly on each flavor, regardless of code changes
-        Utilities.addPeriodicTrigger(newJob, "@daily", true /*always run*/)
+        //Utilities.addPeriodicTrigger(newJob, "@daily", true /*always run*/)
 
         newJob.with {
             logRotator {
@@ -825,11 +825,11 @@ parallel(
                         builder.triggerForBranch(branch)
                         builder.emitTrigger(newJob)
                     }
-                    else {
+                    //else {
                         // Set a push trigger
-                        TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
-                        builder.emitTrigger(newJob)
-                    }
+                    //    TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
+                    //    builder.emitTrigger(newJob)
+                    //}
                 }
             }
         }
-- 
2.7.4

